src/components/Client/ClientOrder.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { orderTypes } from '../../utils/configs';
+import axios from 'axios';
+
+const ClientOrder = () => {
+    let history = useHistory();
+
+    const onSelect = (order) => {
+        axios.post('/client/order', {
+            order: order
+        })
+            .then(function (response) {
+                console.log(response);
+            })
+            .catch(function (error) {
+                console.log(error);
+            });
+        history.push('/taxi')
+    }
+
+    const orderButton = orderTypes.map((order) => {
+        return (
```

Comment:
remove return statement

Reply:
when I remove the return statement, eslint says:
Array.prototype.map() expects a return value from arrow function  array-callback-return